python/ray/data/_internal/execution/interfaces/ref_bundle.py

```diff
@@ -10,6 +10,16 @@
 from ray.types import ObjectRef
 
 
+@dataclass
+class BlockSlice:
+    """A slice of a block."""
+
+    block_index: int
+    start_offset: int
+    end_offset: int
+    output_index: int
```

Comment:
why do you need `block_index/output_index` if a `BlockSlice` is for a single Block?